mobile/src/home-view.js

```diff
@@ -1,10 +1,12 @@
 import React, { Component } from 'react'
-import ReactNative, { AsyncStorage, Modal, Platform, ScrollView, Share, Text, TouchableOpacity, View, Alert } from 'react-native'
+import ReactNative, { AsyncStorage, Modal, Platform, ScrollView, Share, Text, TouchableOpacity, View, Alert, KeyboardAvoidingView, TextInput } from 'react-native'
 import client, { Avatar, TitleBar } from '@doubledutch/rn-client'
 import { LabeledTextInput, FlatButton } from './dd-ui'
 import { CardView, CardListItem, EditCardView } from './card-view'
 import { ScanView, CodeView } from './scan-view'
 import FirebaseConnector from '@doubledutch/firebase-connector'
+import {KeyboardAwareScrollView} from 'react-native-keyboard-aware-scroll-view'
+// Text.defaultProps.allowFontScaling=false
```

Comment:
Are we leaving this in?

Reply:
I will take that out and push this update to production